src/dataDisplay/table/table.d.ts

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+export interface TablePropTypes {
+    /**
+     * Basic (default) styling.
+     */
+    basic?: boolean;
+    /**
+     * If `true`, all columns will be divided.
+     */
+    celled?: boolean;
+    /**
+     * The content of the Table.
+     */
+    children?: React.ReactNode;
+    /**
+     * Override or extend the styles applied to the Table.
+     */
+    classes: {
+        tableSticky: string;
+        tableStickyColumns: string;
+    };
+    /**
+     * Add additional classes to the Table.
+     */
+    className: string;
+    /**
+     * If `true`, applies `width: auto` to the Table.
+     */
+    collapsing: boolean;
+    /**
+     * If `true`, the far left column is divided from the rest of the columns.
+     */
+    definition: boolean;
+    /**
+     * If `true`, columns will be evenly spaced.
+     */
+    fixed: boolean;
+    /**
+     * Change the font size of text in the Table.
+     */
+    fontSize: 'xxlarge' | 'xlarge' | 'large' | 'medium' | 'small' | 'xsmall' | 'xxsmall';
+    /**
+     * Deprecated prop.
+     */
+    fullWidth: boolean;
+    /**
+     * Add an id to the Table.
+     */
+    id: string;
+    /**
+     * Table rows and cells can be highlighted as if they have been selected.
+     */
+    selectable: boolean;
+    /**
+     * Truncate copy within cells in the Table
+     */
+    singleLine: boolean;
+    /**
+     * Change the cells vertical size in the Table.
+     */
+    size: 'l' | 'large' | 'm' | 'medium' | 's' | 'small';
+    /**
+     * Deprecated prop.
+     */
+    stackable: boolean;
```

Comment:
Fixed.

Reply:
The TypeScript declaration was fixed.  Looks like we left the default prop still `null`, but I can be persuaded that's non-blocking.  Especially since this is a deprecated prop.  We probably don't want consumers passing `null`; we probably want them to only specify `true` or `false` or don't specify this prop at all.  And the TypeScript declarations are mostly to help TypeScript consumers.